data-guides/index.md

```diff
@@ -1,20 +1,28 @@
 ---
-title: Welcome to the Data Wrangling Handbook!
+title: Welcome to the Labs Handbook!
 ---
 
-The Data Wrangling Handbook is a companion text to the [`School of Data`](http://schoolofdata.org/). Its function is something like a traditional textbook - it will provide the detail and background theory to support the School of Data courses and challenges.
+The Labs Handbook is a companion text to the [`School of Data`](http://schoolofdata.org/). Its function is something like a 
```

Comment:
This is no longer true and we should delete this paragraph (though we may want to reference the School of Data courses somewhere).